### PR TITLE
Restrict fw dv permissions

### PR DESCRIPTION
**Pull request by smacadam**
This PR is for ticket #176870613 "Data viewers/ Field workers have too many privileges" - editing VA records should be restricted to Data Managers and Admins.

Testing
- Run manage.py initialize_groups to add the new permissions
- Field Worker & Data Viewer users should not see the "Edit Record" button when viewing a record
- Field Worker & Data Viewer should get a 403 if they manually navigate to localhost:5000/va_data_management/edit/2
- Data Managers and Admins should still be able to edit records

### CODE REVIEW

File: va_explorer/templates/va_data_management/show.html [hunk 1]
```diff
@@ -24,7 +24,9 @@
 <div class="tab-content">
   <div class="tab-pane fade show active" id="record" role="tabpanel" aria-labelledby="record-tab">
     <div class="row mt-3">
-      <a class="btn btn-primary" href="{% url 'data_management:edit' id=id %}">Edit Record</a>
+      {% if perms.va_data_management.change_verbalautopsy %}
```
Comment (deastman) [inline va_explorer/templates/va_data_management/show.html:27]: @smacadam Also need to wrap the two buttons at the bottom of the page, i.e.: 

`{% if perms.va_data_management.change_verbalautopsy %} 
          <a class="btn btn-primary mx-2" href="{% url 'data_management:reset' id=id %}">Reset to Original</a></p>
          <a class="btn btn-primary mx-2" href="{% url 'data_management:revert_latest' id=id %}">Revert Most Recent Change</a>
{% endif %} `

File: va_explorer/va_data_management/views.py [hunk 2]
```diff
@@ -68,8 +69,9 @@ def get_context_data(self, **kwargs):
         return context
```
Comment (deastman) [inline va_explorer/va_data_management/views.py:71]: @smacadam Reset and RevertLatest have to be similarly restricted

File: va_explorer/templates/va_data_management/show.html (hunk 1 above)
Comment (smacadam) [reply va_explorer/templates/va_data_management/show.html:27]: Added check

File: va_explorer/va_data_management/views.py (hunk 2 above)
Comment (smacadam) [reply va_explorer/va_data_management/views.py:71]: added permissions and tests

**Comment by deastman** [comment]
@smacadam Tested! This all looks good to me. Are you waiting on Pete's PR before merging this?

**Comment by smacadam** [comment]
Yes waiting on Pete's, there might be a few conflicts to resolve.

**Review by ajbarnes — COMMENTED** [review]
Code review looks good pending the one potential conflict with #42 

After pulling the changes locally I was also able to visually confirm expected functionality from suggested tests along with a successful pytest including the new tests.

After discussing #42 I say LGTM.

**Review by ajbarnes — APPROVED** [review]

**Review by ajbarnes — APPROVED** [review]

**Pull request merged**